src/Opdex.Platform.WebApi/Controllers/IndexController.cs

```diff
@@ -66,7 +77,7 @@ public async Task<IActionResult> ResyncFromDeployment(ResyncFromDeploymentReques
             var markets = await _mediator.Send(new RetrieveAllMarketsQuery(), cancellationToken);
             if (markets.Any())
             {
-                throw new Exception("Markets exist");
+                throw new NotFoundException("No markets exist to resync.");
```

Comment:
Consider changing this exception to explain that markets _do_ exist so their request is invalid. 

Maybe a bad request with "Markets already deployed" message.

Reply:
Sorted